src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -69,17 +75,78 @@ export class FeaturedArtists extends React.Component<FeaturedArtistsProps, Featu
     return allArtists
   }
 
+  failedFollowChange() {
+    // callback for analytics purposes
```

Comment:
The tracking calls are still to come, correct? (It's ticketed elsewhere, so just double-checking)

Reply:
Yes, we still need to add tracking per what @abhitip outlined in the Jira ticket!